### PR TITLE
Add contributing guidelines, move contributors

### PR DESCRIPTION
**Pull request by GerbrichFerdinands**
Hi! I've made some changes on contributing/contributors. 
I hope this leads the way for new contributors to ASReview!

This is what I did: 
- move contributors to contributors.md
- add contributing guidelines 
- add two issue templates:
  - bug report
  - feature request 

Looking forward to your feedback.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Contributing to ASReview
+Hello dear contributor-to-be! :wave:
```
Comment (J535D165) [inline contributing.md]: Nice!

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+name: "\U0001F680 Feature Request"
+about: "I have a suggestion (and may want to implement it \U0001F642)!"
+title: ''
+labels: 'i: enhancement, i: needs triage'
```
Comment (J535D165) [inline .github/ISSUE_TEMPLATE/feature-request.md]: We don't use the label ` i: needs triage`

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+We'd love to collaborate with you!
+Below you will find some guidelines for contributing to ASReview.
+
+#### Feel free to propose changes of any kind!
+Your ideas and input are very much appreciated.
+From contributing to source code to improving readability of the documentation, all suggestions are welcome!
+
+There are multiple ways to contribute:
+- [Reporting bugs](#reporting-bugs) :bug: :ant: :beetle:
+- [Suggesting enhancements](#feature-request)
```
Comment (J535D165) [inline contributing.md]: A line to suggest interesting datasets with a link to the datasets repo.

Comment (J535D165) [reply contributing.md]: Template over there might be interesting as well :)

Comment (J535D165) [reply contributing.md]: My bad. This one is already there!!

File: contributors.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- Gerbrich Ferdinands ([@GerbrichFerdinands](https://github.com/GerbrichFerdinands))
+- [Qixiang Fang](https://www.uu.nl/staff/QFang) ([@fqixiang](https://github.com/fqixiang))
+- Albert Harkema ([@sasafrass](https://github.com/sasafrass))
+- Joukje Willemsen ([@JoukjeWillemsen](https://github.com/JoukjeWillemsen))
+- Maarten Hoogerwerf
+- Maarten Meijer ([@mmeijer23](https://github.com/mmerijer23))
+- D W J Visser
+- Martijn Huijts ([@MartijnUX](https://github.com/MartijnUX))
+- Sybren Hindriks
+- Yongchao Terry Ma ([@terrymyc]([@MartijnUX](https://github.com/terrymyc))
```
Comment (qubixes) [inline contributors.md]: There's an extra Martijn here.

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- Propose a fix to an existing issue. :warning:
+  - See the [issue list here](https://github.com/asreview/asreview/issues).
+- Add a new dataset to [our open dataset repository](https://github.com/asreview/systematic-review-datasets/pull/11)
+  - See this [pull request](https://github.com/asreview/systematic-review-datasets/pull/11)) for an example.
+
+## First timer?
+Do you have little experience with contributing to open-source project and working with GitHub? :octocat: No worries! There are some amazing tutorials out there:
+
+- [First timers only](https://www.firsttimersonly.com)
+  - See the [firstcontributions repo](https://github.com/firstcontributions/first-contributions) for a tutorial on how to contribute code.
```
Comment (qubixes) [inline contributing.md]: Put a dash between first and contributions?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+your input code here
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Version information (please complete the following information):**
```
Comment (qubixes) [inline .github/ISSUE_TEMPLATE/bug.md]: We want the complete all the information, no? So probably not necessary to put it here.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+1.
+2.
+3.
+
+
+```
+your input code here
+```
+
+**Expected behavior**
```
Comment (qubixes) [inline .github/ISSUE_TEMPLATE/bug.md]: Is this one necessary? Should be clear from the bug description.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug with the front-end is.
+
+**To Reproduce**
+Steps to reproduce the problem:
+
+1.
```
Comment (qubixes) [inline .github/ISSUE_TEMPLATE/bug.md]: I think we can leave out the the 1., 2., etc.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+name: "Bug report \U0001F41B" 
+about: "If something isn't working as expected \U0001F914."
+title: ''
+labels: bug
+assignees: J535D165, qubixes
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug with the front-end is.
```
Comment (qubixes) [inline .github/ISSUE_TEMPLATE/bug.md]: Is this only for the front-end?

**Comment by qubixes** [comment]
I'm personally *not* a fan of the style in the contributing file, but then again, I'm just a grumpy old man, so what gives?

**Comment by GerbrichFerdinands** [comment]
@qubixes your honest opinion is more than welcome 👴

File: contributors.md (hunk 4 above)
Comment (Rensvandeschoot) [reply contributors.md]: @GerbrichFerdinands can you check with all persons if they want to be mentioned here?

Comment (GerbrichFerdinands) [reply contributors.md]: Great remark, I will send everyone an e-mail today!

File: .github/ISSUE_TEMPLATE/bug.md (hunk 9 above)
Comment (GerbrichFerdinands) [reply .github/ISSUE_TEMPLATE/bug.md]: Huh, the "with the frond-end" part shouldn't be there. Thanks for your attentiveness!

File: .github/ISSUE_TEMPLATE/bug.md (hunk 7 above)
Comment (GerbrichFerdinands) [reply .github/ISSUE_TEMPLATE/bug.md]: I agree that ideally this should be clear from the bug description. 
I still think it is nice to explicitly ask for providing both "what is wrong" (the bug) and "what you think should have happened ("expected behaviour").

This helps preventing misunderstandings about the functionality of the code. For example, the conversation we had last week about the "--abstract_only" option only made sense after we found out I was assuming the option did something different than it was actually doing.

I removed this separate heading on expected behaviour, and added a sentence on it under the bug description heading, c65c5182bf98a0cefc18ccb125fa50c0cd1a43b8. What do you think?

File: .github/ISSUE_TEMPLATE/user-interface.md [hunk 10]
```diff
@@ -17,13 +18,11 @@ Steps to reproduce the behavior:
...
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
 
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Browser (please complete the following information):**
+**Version information**
  - OS: [e.g. Windows]
```
Comment (qubixes) [inline .github/ISSUE_TEMPLATE/user-interface.md]: Browser version / ASReview version?

**Comment by J535D165** [comment]
Veel dank 👍

**Pull request merged**